addon/bool.js

```diff
@@ -0,0 +1,3 @@
+import lazyCurriedComputed from 'ember-macro-helpers/lazy-curried-computed';
+
+export default lazyCurriedComputed((get, key) => !!get(key));
```

Comment:
This can be regular `curriedComputed` because there is only one key we have to worry about.